packages/console/src/pages/SignInExperience/tabs/SignUpAndSignInTab/components/SignInMethodEditBox/utilities.ts

```diff
@@ -0,0 +1,49 @@
+import { SignInIdentifier } from '@logto/schemas';
+
+import { SignInMethod } from './types';
+
+export const mutateVerificationState = (
```

Comment:
nit: have a better name? that sounds like an "action" function but it returns a computed result.

Reply:
updated.